lib/SILGen/SILGen.cpp

```diff
@@ -1540,6 +1513,36 @@ void SILGenModule::emitAbstractFuncDecl(AbstractFunctionDecl *AFD) {
         SILDeclRef::BackDeploymentKind::Thunk);
     emitBackDeploymentThunk(thunk);
   }
+
+  // Emit differentiability witness for the function referenced in
+  // @derivative(of:) attribute registering current function as VJP / JVP.
+  for (auto *derivAttr : AFD->getAttrs().getAttributes<DerivativeAttr>()) {
+      auto *F = getFunction(SILDeclRef(AFD), NotForDefinition);
```

Comment:
Nit: it looks like that the code mostly uses `camelCase`, so it's probably better to name this `f` or `func` or smth instead of `F`